### PR TITLE
sci-mathematics/gimps: Version bump to gimps-30.3.6

### PR DESCRIPTION
**Pull request by PPed72**
Also switched source to https://

Signed-off-by: Paolo Pedroni <paolo.pedroni@iol.it>
Closes: https://bugs.gentoo.org/746911
Package-Manager: Portage-3.0.4, Repoman-3.0.1

### CODE REVIEW

File: sci-mathematics/gimps/gimps-30.3.6.ebuild [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+	einstalldocs
+
+	readme.gentoo_create_doc
+
+	newinitd "${FILESDIR}/${PN}-28.9-init.d" gimps
+	newconfd "${FILESDIR}/${PN}-25.6-conf.d" gimps
+
+	systemd_dounit "${FILESDIR}/${PN}.service"
+	systemd_newtmpfilesd "${FILESDIR}/${PN}.tmpfiles" "${PN}.conf"
```
Comment (juippis) [inline sci-mathematics/gimps/gimps-30.3.6.ebuild:59]: You should use *tmpfiles.eclass* for this task.

File: sci-mathematics/gimps/gimps-30.3.6.ebuild [hunk 2]
```diff
@@ -0,0 +1,71 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit pax-utils systemd readme.gentoo-r1
```
Comment (juippis) [inline sci-mathematics/gimps/gimps-30.3.6.ebuild:6]: Also please sort this while at it, you need to inherit *tmpfiles.eclass* and use its function here.

File: sci-mathematics/gimps/gimps-30.3.6.ebuild (hunk 1 above)
Comment (PPed72) [reply sci-mathematics/gimps/gimps-30.3.6.ebuild]: Done.

File: sci-mathematics/gimps/gimps-30.3.6.ebuild (hunk 2 above)
Comment (PPed72) [reply sci-mathematics/gimps/gimps-30.3.6.ebuild]: Done.

**Comment by PPed72** [comment]
I think I've addressed all concerns. Awaiting for merge (or further comments, of course)

File: sci-mathematics/gimps/Manifest [hunk 3]
```diff
@@ -1,2 +1,4 @@
...
 DIST p95v298b6.linux64.tar.gz 7032597 BLAKE2B de4de01d68caba4530d5adca165bfaa28cdb7e276b44355b200245cac6bab80bc05885a994c06331c2f66bdcd236c8bdb6cc8206ca40fb583bce6dc98eced7b1 SHA512 cb0ea195f4167e22bfdeef02872205a40ccd4d110abc9c354a413284db9aef1dbf9888ecdc3f09262e0d1033c0e4bed1f23d4d966fb58c437673a81095252bd1
+DIST p95v303b6.linux32.tar.gz 5702312 BLAKE2B 728c8b8e7e2b5187dc8126bccc5e2c165fbfbc0900d6c35517cd21b78e5292df143641e3766cc0146918062039b1eecab568f9b278f64b1b9197ee2f775603b9 SHA512 3e06fb77be07334949ece3ae152bdf5491f0b7f8e33e0e8ed2af3c34332841975a9514d1993ed2355abfa4e09f93e42093d02f96dc30c3471ae9f974cc2de11a
+DIST p95v303b6.linux64.tar.gz 7046127 BLAKE2B 9cde9c2670203071cfc463434a47c8123064cb232bfeb25d8767cad2c25ea75b395e2ec135b7b4a6dc9616e9d6b05ead371e1083274a82468f62a0e1de93c98f SHA512 6d6ded9a1724993d42adee28d400b0337835c52a4ce42271e62efb37134adfc6f017f55475860fbc1224e377a227ef8547b8852a35c082c37e996f4a1c1b570c
```
Comment (juippis) [inline sci-mathematics/gimps/Manifest:4]: Ideally we'd want to rename these distfiles, maybe next time...

**Review by juippis — APPROVED** [review]

**Pull request closed without merging**